lib/round-tracker.js

```diff
@@ -225,3 +237,27 @@ async function defineTasksForRound (pgClient, sparkRoundNumber) {
     TASKS_PER_ROUND
   ])
 }
+
+/**
+ * @param {Awaited<ReturnType<import('./ie-contract.js').createMeridianContract>>} contract
+ * @param {bigint} roundIndex
+ * @returns {Promise<biging>} Filecoin Epoch (ETH block number) when the SPARK round started
```

Comment:
```suggestion
 * @returns {Promise<bigint>} Filecoin Epoch (ETH block number) when the SPARK round started
```

Reply:
It's time to setup TypeScript checks in this repo too 💪🏻